### ISSUE TITLE
[Adaptive Card > InputFormwithRTL.Json]: Screen Reader is announcing the Button role information twice.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#176]
### Target Platforms

iOS

### SDK Version

Version 1.0 (2.4.0-beta.24.5.30.1)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.24.5.30.1)
Screen Reader: Voice Over

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "InputFormWithRTL.json" control and double tap to activate it.
7. Perform right swipe and navigate to the 'Arabi' card button and double tap to expand it.
8. Navigate to the button using swipe gesture and observe the issue.

**Actual Result:**
While navigating to the button present under 'Arabi', screen reader is conveying the role information twice. it is announcing as **'Ukadim Button Button'**

**Expected Result:**
Screen Reader should not announce the role information twice. It should announce the button role only one time and it should announce as **'Ukadim button.'**.

**User Experience:**
It will impact the screen reader user as they will get confused about the information conveyed by the screen reader if the role information conveyed twice.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/ba40a4bf-7f9c-4206-9b78-43de589635b4

### Card JSON

```JSON
NA
```
